docker-stack.yml

```diff
@@ -69,6 +69,7 @@ services:
             - sauber-network
         volumes:
             - db_data:/var/lib/postgresql/data
+            - ./db_backups:/home/backups/database/postgresql
```

Comment:
Is there a reason why it is not a volume as `db_data` ? Just curious, but looks good to me.

Reply:
If the DB container fails to run, restart - have the backups ready on the host filesystem. Avoid e.g. having to mount the backup volume on a helper-container, etc.